vars/default.yml

```diff
@@ -0,0 +1,5 @@
+---
+__vpn_packages:
+  - libreswan
+
+__vpn_ikev2: 'insist'
```

Comment:
see above - if you use the idiom I suggested then you do not need to have a `vars/default.yml` and you can put these in `vars/main.yml`